api/src/main/java/io/serverlessworkflow/api/mapper/BaseObjectMapper.java

```diff
@@ -31,6 +33,11 @@ public BaseObjectMapper(JsonFactory factory, WorkflowPropertySource workflowProp
 
     configure(SerializationFeature.INDENT_OUTPUT, true);
     registerModule(workflowModule);
+    configure(SerializationFeature.WRITE_EMPTY_JSON_ARRAYS, false);
```

Comment:
this change is relevant to the fix?  seems to be a rebase from another bug fix that @tsurdilo  fixed recently

Reply:
@manick02  Yes, this was not related to the fix. I will create a new PR to avoid confusion.